scripts/environment.sh

```diff
@@ -78,6 +78,13 @@ if [[ ${GINKGO_FOCUS:-} == "integration" || ${GINKGO_FOCUS:-} == "basic" ]]; the
   export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-"1"}
 fi
 
+# IPReuse feature test environment vars and config
+if [[ ${GINKGO_FOCUS:-} == "features" && ${GINKGO_SKIP:-} == "pivoting remediation" ]]; then
```

Comment:
This looks weird. "GINKGO_FOCUS == features" would render "GINKGO_SKIP" unnecessary. Did you mean to use || instead of &&?

Reply:
We following tests under GINKGO_FOCUS == features

- [ip-reuse] [features]
- [healthcheck] [remediation] [features]
- [pivoting] [features]
- [remediation] [features]

to set env variables only for IPReuse I need to make sure GINKGO_FOCUS == features && GINKGO_SKIP == pivoting remediation
-e2e-feature-test-main-features is only running IPReuse, perhaps we should change the name of the test.